corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -46,8 +60,17 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                 value = values;
             } else if (values.length === 1) {
                 value = values[0];
-                if (model.get("input") === "daterange") {
-                    value = value.replace("__range__", "").replace("__", separator);
+                if (model.get("input") === "date") {
+                    value = toUiDate(value);
+                } else if (model.get("input") === "daterange") {
+                    // Take sticky value ("__range__2023-02-14__2023-02-17")
+                    // or default value ("2023-02-14 to 2023-02-17")
+                    // coerce to "02/14/2023 to 02/17/2023", as used by the widget
```

Comment:
You could argue that we should try to move the default value format to be consistent with one of the other two, but I think it's reasonable for us to be flexible in what input we accept (and this lets us skip a migration)

Reply:
Consistency is nice, but I think I would agree given that the date format "standard" can vary so much.